src/server/snapshot.cc

```diff
@@ -364,6 +364,14 @@ size_t SliceSnapshot::GetTotalChannelCapacity() const {
   return dest_->GetSize();
 }
 
+size_t SliceSnapshot::GetTempBuffersSize() const {
```

Comment:
I think in this context "temp buffer" is an implementation detail. Maybe call it "MemoryUsed" instead?

Reply:
Now I see there are three functions in SliceSnapshot and it's not easy to understand how they are different in semantics.
nit: I would prefer that `SerializerBase::GetTempBuffersSize` name will be different from that of SliceSnapshot

Reply:
Removed the term `Total` and added a comment :pray: